src/components/panel/demoGroups/script.js

```diff
@@ -0,0 +1,95 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('panelGroupsDemo', ['ngMaterial'])
+    .controller('PanelGroupsCtrl', PanelGroupsCtrl)
+    .controller('PanelMenuCtrl', PanelMenuCtrl);
+
+  function PanelGroupsCtrl($mdPanel) {
+    this.showMenu = showMenu;
+
+    this.settings = [
+      'Home',
+      'About',
+      'Contact'
+    ];
+    this.favorite = [
+      'Add to Favorites'
+    ];
+    this.more = [
+      'Account',
+      'Sign Out'
+    ];
+    this.items;
+
+    function showMenu($event, type, group) {
+      if (type === 'settings') {
+        this.items = this.settings;
+      } else if (type === 'favorite') {
+        this.items = this.favorite;
+      } else {
+        this.items = this.more;
+      }
+
+      var template = '' +
+          '<div class="menu-panel">' +
+          '  <div class="menu-content">' +
+          '    <div class="menu-item" ' +
+          '        ng-repeat="item in ctrl.items">' +
+          '      <button class="md-button">' +
+          '        <span>{{item}}</span>' +
+          '      </button>' +
+          '    </div>' +
+          '    <div class="menu-divider"></div>' +
+          '    <div class="menu-item">' +
+          '      <button class="md-button" ng-click="ctrl.closeDialog()">' +
+          '        <span>Close Menu</span>' +
+          '      </button>' +
+          '    </div>' +
+          '  </div>' +
+          '</div>';
+
+      var position = $mdPanel.newPanelPosition()
+          .relativeTo($event.srcElement)
+          .addPanelPosition(
+            $mdPanel.xPosition.ALIGN_START,
+            $mdPanel.yPosition.BELOW
+          );
+
+      $mdPanel.newPanelGroup(group, {
+        maxOpen: 2
+      });
+
+      var config = {
+        id: group + '_' + type,
+        attachTo: angular.element(document.body),
+        controller: PanelMenuCtrl,
+        controllerAs: 'ctrl',
+        template: template,
+        position: position,
+        panelClass: 'menu-panel-container',
+        locals: {
+          items: this.items
+        },
+        openFrom: $event,
+        clickOutsideToClose: false,
+        escapeToClose: false,
+        focusOnOpen: false,
+        zIndex: 100,
+        propagateContainerEvents: true,
+        groupName: group
+      };
+
+      $mdPanel.open(config);
+    }
+  }
+
+  function PanelMenuCtrl(mdPanelRef) {
+    this.closeDialog = closeDialog;
+
+    function closeDialog() {
+      mdPanelRef && mdPanelRef.close();
```

Comment:
@ErinCoughlan I do not think so. Destroying a panel makes it unable to be re-opened, and in this demo, I am applying an ID to each opened panel, which means it can be closed and re-opened from the click event.